boards/arm/mimxrt1064_evk/mimxrt1064_evk.dts

```diff
@@ -62,13 +62,18 @@
 	};
 };
 
+arduino_i2c: &i2c1 {};
+
 &lcdif1 {
 	status = "ok";
 	port {
 		lcd_panel_out: endpoint {
 			remote-endpoint = <&lcd_panel_in>;
 		};
 	};
+
+&i2c1 {
+	status = "ok";
```

Comment:
This seems slightly off, since you are adding '{' seems like the change should be adding the close '}'